### PR TITLE
update polkadot version to v0.9.19

### PR DESCRIPTION
**Pull request by itsmemz**

### CODE REVIEW

File: runtime/frost/src/lib.rs [hunk 1]
```diff
@@ -417,13 +417,11 @@ parameter_types! {
 }
 
 impl pallet_transaction_payment::Config for Runtime {
-	type OnChargeTransaction = CurrencyAdapter<Balances, DealWithFees>;
```
Comment (mohamedelshami) [inline runtime/frost/src/lib.rs:420]: This is undoing the required change for fee handling. Please line from previous commit

File: node/src/cli.rs [hunk 2]
```diff
@@ -20,57 +25,59 @@ pub struct Cli {
 
 
 /// Possible subcommands of the main binary.
-#[derive(Debug, clap::Subcommand)]
+#[derive(Debug, Parser)]
```
Comment (mohamedelshami) [inline node/src/cli.rs:28]: Assuming this is fine? why clap::Subcommand is replaced?

File: node/src/command.rs [hunk 3]
```diff
@@ -200,13 +199,54 @@ pub fn run() -> Result<()> {
...
-            let chain_spec = &runner.config().chain_spec;
-            if chain_spec.is_arctic() {
-                runner.sync_run(|config| cmd.run::<arctic_runtime::Block, arctic_service::Executor>(config))
-            } else {
-                runner.sync_run(|config| cmd.run::<frost_runtime::Block, frost::ExecutorDispatch>(config))
-            } 
+			let runner = cli.create_runner(cmd)?;
+			let chain_spec = &runner.config().chain_spec;
+            let is_arctic = chain_spec.is_arctic();
+            match cmd {
```
Comment (mohamedelshami) [inline node/src/command.rs:205]: I see, all this chunk of code was missing for benchmark support?

File: runtime/arctic/src/lib.rs [hunk 4]
```diff
@@ -920,6 +917,7 @@ impl_runtime_apis! {
 				None
 			};
 
+			let is_transactional = false;
```
Comment (mohamedelshami) [inline runtime/arctic/src/lib.rs:920]: Was this always default to false before?

**Review by mohamedelshami — COMMENTED** [review]
I've made a few comments please review. Also, the version is v0.0.19, not v19

File: runtime/arctic/src/lib.rs (hunk 4 above)
Comment (itsmemz) [reply runtime/arctic/src/lib.rs:920]: Yes, this is an extra argument needed to satisfy, I have copied it from frontier node template

File: node/src/command.rs (hunk 3 above)
Comment (itsmemz) [reply node/src/command.rs:205]: This added newly with polkadot v19 on frontier node template

File: node/src/cli.rs (hunk 2 above)
Comment (itsmemz) [reply node/src/cli.rs:28]: Yes

File: runtime/frost/src/lib.rs (hunk 1 above)
Comment (itsmemz) [reply runtime/frost/src/lib.rs:420]: Ok sure

**Pull request merged**